README.md

```diff
@@ -46,6 +46,8 @@ Then, you can run:
 Start-BitsTransfer -Source https://git.io/rustlings-win -Destination $env:TMP/install_rustlings.ps1; Unblock-File $env:TMP/install_rustlings.ps1; Invoke-Expression $env:TMP/install_rustlings.ps1
 ```
 
+When you get a permission denied message then you have to exclude the directory where you placed the rustlings in your virus-scanner.
```

Comment:
Hi @RoelofWobben, can you please move this sentence after line 51? The note should come after the installation guide.

Thanks,
Abdou

Reply:
I hope this is a better place otherwise said exactly where you want it to be.
This is  a little vague after line 51,

Reply:
I am sorry about the confusion. The line numbering I was looking at did not make any sense. I meant that you should place the text in the **Windows** section right below `To install Rustlings. Same as on MacOS/Linux, you will have access to the rustlings command after it.`

That should be a better place for it. Sorry about the unnecessary confusion.